.travis.yml

```diff
@@ -32,6 +33,8 @@ script:
 
 after_success:
   # - coverage run test/tests.py && codeclimate-test-reporter --file .coverage --token $CODECLIMATE_REPO_TOKEN
+  - coverage run --source=. -m unittest discover -s test/
```

Comment:
It might be a good idea to put these commands in a script/test bash file and invoke that file in travis.yml. Done that way, we can also use the script during development as well.

Reply:
Please see the conventions of GitHub's [one script to rule them all](https://github.com/github/scripts-to-rule-them-all) for reference.

Reply:
install.sh should also move to script then.

Reply:
Yup. That would be appropriate. Also we need to rename it to install (just to stick to naming conventions of GitHub)